### ISSUE TITLE
Does this require a locally installed `git`?

### ISSUE THREAD

**Issue by huy-vuong** [kevmoo/git#24]
I'm interested in using this for mobile and web if it can write to repos (`commit`, `push`, etc.). If I wanted to use this on Android or so, do I need to set anything up in advance to make it work?

**Comment by Marty** [comment]
Yes it does, see [top_level.dart#L13](https://github.com/kevmoo/git/blob/master/lib/src/top_level.dart#L13).
